examples/general_demo.py

```diff
@@ -41,7 +44,7 @@ def main():
     phase_e2 = nii_phase_e2.get_fdata() / 4096.0 * 2.0 * np.pi - np.pi
```

Comment:
I've seen this scaling in some other files and functionality. It is a detail, but I think it would be good to make an utility function that scales. This way, it will be easier to update our code in the futur. Also, I don't think it is a good idea to explicitly scale phases in the example since it can be prone to errors from the user part:
```suggestion
    phase_e2 = scale_phase(nii_phase_e2.get_fdata(), [-np.pi, np.pi])
    
def scale_phase(phase_data, range)
    return np.substract(np.multiply(np.divide(phase_data, 4096.0), 2.0*np.pi), np.pi)
    #or return np.interp(-np.pi, np.pi, phase_data)
```
The use of numpy functions is generally faster for large arrays and implements a true division which adapts the output format for better precision instead of the floor division in Python.

Reply:
Used `np.interp(nii_phase_e2.get_fdata(), [0, 4096], [-np.pi, np.pi])` but removed the scale_phase function since it added a layer of function call without removing any line of code in the script. `interp` is also self descriptive.